src/components/empty_prompt/empty_prompt.stories.tsx

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React from 'react';
+import type { Meta, StoryObj } from '@storybook/react';
+
+import { EuiEmptyPrompt, EuiEmptyPromptProps } from './empty_prompt';
+
+import { EuiButton, EuiButtonEmpty } from '../button';
+import { EuiTitle } from '../title';
+import { EuiLink } from '../link';
+import { EuiImage } from '../image';
+import { EuiPageTemplate } from '../page_template';
+import illustration from '../../../src-docs/src/images/empty-prompt/illustration.svg';
+
+const meta: Meta<EuiEmptyPromptProps> = {
+  title: 'EuiEmptyPrompt',
+  component: EuiEmptyPrompt,
+  argTypes: {
+    element: { control: 'text' },
+    body: {
+      control: 'text',
+    },
```

Comment:
[nit] let's make both arg types use a more compressed syntax
```suggestion
    element: { control: 'text' },
    body: { control: 'text' },
```

Reply:
Actually let's go ahead and remove the `body` control completely. I don't think this needs configuration personally, it flips just fine between text vs JSX.
```suggestion
    element: { control: 'text' },
```

Reply:
Should `body` in the playground remain a string? Or should we move it back to a `<p>` since the Page Template story is using multiple paragraphs?